src/Core/src/Platform/Android/KeyboardExtensions.cs

```diff
@@ -60,5 +65,67 @@ public static InputTypes ToInputType(this Keyboard self)
 
 			return result;
 		}
+
+		internal static bool HideKeyboard(this AView inputView)
+		{
+			var focusedView = inputView.Context?.GetActivity()?.Window?.CurrentFocus;
+			AView tokenView = focusedView ?? inputView;
```

Comment:
From what @PureWeen and I tested, the focusedView logic here doesn't seem to change anything. Let's just use the inputView here

```suggestion
			AView tokenView = inputView;
```